### ISSUE TITLE
Unable to run app using flutter_scene (using CMake)

### ISSUE THREAD

**Issue by AndyPandy25** [bdero/flutter_scene#44]
So, I have tried to build my app that has flutter_scene (and some imports needed with that), but have been getting this error mesage:
Unhandled exception:
Exception: CMake generate step failed (exit code 1):
STDERR: CMake Error at CMakeLists.txt:2 (project):
  Running

   'nmake' '-?'

  failed with:

   no such file or directory

CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage

STDOUT: -- Configuring incomplete, errors occurred!

Now, CMake is installed, and it is added to my path. As well as Visual Studio with C++ (I am using Windows)
I have not been able to locate anyone having this same problem as I am so I decided to add this issue to see if anyone knows how to fix this.

**Comment by bdero** [comment]
Hi, nmake ships with Visual Studio and should be available in the Visual Studio Developer Command Prompt environment.

- Which terminal are you using?
- Which version of flutter_scene are you using?
- What command(s) are you using to kick off the build/run?
- Which version of Visual Studio are you using?

**Comment by AndyPandy25** [comment]
I am trying to build the app using Android Studio, Visual Studio 2022. And flutter_scene version is the latest one, 0.7 I believe it is

**Comment by bdero** [comment]
My guess would be that the MSVC build tools aren't in your path when building via Android Studio.

First, make sure the build tools are actually installed -- nmake should be located in a directory similar to: `C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.39.33519\bin\Hostx64\x64`, where the specific MSVC version may differ.
The tool should also be in the path of the VS dev command prompt.

If it's not installed, you can use the [VS 2022 installer](https://learn.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-170#step-4---choose-workloads) to add it.

Once you have it, add the MSVC x64 bin directory to your system path. If it still doesn't work from Android Studio, try using `flutter run` from the Visual Studio Developer Command Prompt.

**Comment by AndyPandy25** [comment]
I checked and I did not have that installed, so I installed it. I also added it to my Path, and tried to build it in Android Studio, which failed, same error as before. 

Afterwards, I tried with the Visual Studio Developer Command Prompt as you suggested, and that did the trick. Appreciate it!
Still had some errors when compiling, due to incompatibility of versions between some of my imports, but I fixed them and it compiles!
